staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -339,12 +359,6 @@ var (
 					[]string{"endpoints", "configmaps"},
```

Comment:
here, aren't we effectively saying that every configmap crud is treated as a leader election operation?

Reply:
Well, every one from a service account.  And endpoint CRUD too.  The problem is that out-of-tree code can still do leader elections this way, and the earlier discussion seemed to settle on the idea that treating all this CRUD as leader election is less bad than treating the actual leader elections as regular workload.

Reply:
my concern is taking "all" configmap operations and giving it a dedicated priority level `leader-election` with a concurrency share of only `10`. configmap operations are very frequent if i am not mistaken.